index.html

```diff
@@ -707,6 +707,10 @@ <h2>
     It is recommended to do so before {{ComputePressureObserver/disconnect()}}, 
     otherwise {{ComputePressureObserver/disconnect()}} will clear them and they will be lost forever.
   </p>
+  <p>
+    For example, we might want to check the pressure while loading some benchmarking/test workload.
+    When the task is completed, we want to disconnect quickly, but we are still interested in the last entries provided.
```

Comment:
s/quickly/immediately

When the task is completed, you want to disconnect immediately but still know the values of any pending state changes